tutorbackup/templates/backup/build/backup/backup_services.py

```diff
@@ -21,6 +21,8 @@
 MONGODB_DUMPDIR = os.path.join(DUMP_DIRECTORY, 'mongodb_dump')
 CADDY_DUMPDIR = os.path.join(DUMP_DIRECTORY, 'caddy')
 
+MYSQL_DBS_LIST_FILE = os.path.join(DUMP_DIRECTORY, 'mysql_dbs.txt')
```

Comment:
I'm not 100% sure why we need a separate file here. Can you elaborate on what prompted you do it like this?

Reply:
Certainly there are different ways to implement this but the schemas list on a file was the first thing that came in my mind actually, as I had used this approach before in the past.